src/operator-docs/utility/delayWhen.ts

```diff
@@ -1,6 +1,94 @@
 import { OperatorDoc } from '../operator.model';
 
 export const delayWhen: OperatorDoc = {
-  'name': 'delayWhen',
-  'operatorType': 'utility'
+  name: 'delayWhen',
+  operatorType: 'utility',
+  signature:
+    'public delayWhen(delayDurationSelector: Function, subscriptionDelay?: Observable): Observable',
+  parameters: [
+    {
+      name: 'delayDurationSelector',
+      type: 'Function',
+      attribute: '',
+      description:
+        'A function that returns an Observable for each value emitted by the source Observable'
+    },
+    {
+      name: 'subscriptionDelay',
+      type: 'Observable',
+      attribute: '',
+      description:
+        'An Observable that triggers the subscription to the source Observable once it emits any value.'
+    }
+  ],
+  marbleUrl: 'http://reactivex.io/rxjs/img/delayWhen.png',
+  shortDescription: {
+    description:
+      'Delays the emission of items from the source Observable by a given time span ' +
+      'determined by the emissions of another Observable. <span class="informal">It\'s like  <a href="/#/operators/delay">delay</a>, but ' +
+      'the time span of the delay duration is determined by a second Observable.</span>',
+    extras: []
+  },
+  walkthrough: {
+    description: `
+      <p>
+        delayWhen allows you to 'delay' the emitions on items based on complex criteria you can have,
+        and give you use each item (of the source observable) as an argument to your delayFunction.
+        If the second argument is applied, subscriptionDelay (an Observable) it delays the subscription
+        once it emmits any value.
+      </p>
+    `
+  },
+  examples: [
+    {
+      name: 'It delays the emission of items by 1 seconds',
+      code: `
+          import { Observable } from 'rxjs/Observable';
+          import 'rxjs/add/observable/fromEvent';
```

Comment:
Hi @luillyfe . Thanks for all your PRs, really appreciate it. Could you also change these to es6 imports.